### DISCUSSION TITLE
Select line(s) from Special Key as Android Selection

### DISCUSSION THREAD

**Discussion by malvarco** [gsantner/markor#2344] [Ideas] [answered]
Congratulations for such great App. I use it everyday.
I would like to ask if it would be possible to add the selection from Special Key from Markor ribbon as Android Selection.
![00](https://github.com/gsantner/markor/assets/4598007/b60bf748-ef50-4cb2-b377-f34855587a9d)
![01](https://github.com/gsantner/markor/assets/4598007/efc20d3a-b18d-4bfa-ad9c-405d7a108078)
When I chose "Select Line(s)", lines are selected:
![02](https://github.com/gsantner/markor/assets/4598007/9950bf49-a7f0-4ff4-b094-8932b4832f71)
However, I would like that this selection could be managed by Android's clipboard options as such:
![03](https://github.com/gsantner/markor/assets/4598007/3b69cbc2-3b30-4ab8-b021-644e9334b76e)
So, I could have access to modify the selection or use Android's options to copy and cut the selection.

**Comment by gsantner** [comment, marked as answer]
If somebody is interested, improvements are welcome.
